### PR TITLE
Handle python3.14 ChildWatcher changes

### PR DESCRIPTION
**Pull request by zmedico**
Bug: https://bugs.gentoo.org/941955

### CODE REVIEW

File: lib/portage/util/_eventloop/asyncio_event_loop.py [hunk 1]
```diff
@@ -6,12 +6,22 @@
...
-    from asyncio.unix_events import PidfdChildWatcher
+    from asyncio.unix_events import _ThreadedChildWatcher as ThreadedChildWatcher
 except ImportError:
-    PidfdChildWatcher = None
+    try:
+        from asyncio.unix_events import ThreadedChildWatcher
+    except ImportError:
+        ThreadedChildWatcher = None
+
+try:
```
Comment (thesamesam) [inline lib/portage/util/_eventloop/asyncio_event_loop.py:18]: Our minimum Python is 3.9, I think, so do we need this conditional? Looks OK otherwise.

Comment (thesamesam) [reply lib/portage/util/_eventloop/asyncio_event_loop.py:18]: I think the same might apply to https://github.com/gentoo/portage/commit/7218c9cea6185c82fbc1719489a2d8e814d05ff1.

Comment (zmedico) [reply lib/portage/util/_eventloop/asyncio_event_loop.py:15]: Yes on both accounts, so updated to remove the unnecessary conditionals. Thanks!

**Review by thesamesam — APPROVED** [review]
Thanks!

**Pull request merged**